src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
```

Comment:
you don't need to add health in constructor

Reply:
agree, fix @GarryRocksman comment
health won't be changed for all animals, it'll be 100 for all of them